api/src/pcapi/core/finance/api.py

```diff
@@ -1942,6 +1942,19 @@ def get_granted_deposit(
     return None
 
 
+def do_recredit(user: users_models.User, deposit: models.Deposit) -> models.Recredit | None:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏`do_recredit` c'est un peu redondant. `recredit` suffit
On peut être lus précis avec `recredit_user` par exemple

Reply:
D'ailleurs on dirait que cette fonction est très locale à ce fichier, on peut la marquer comme privée `_recredit_user`